commitlint.config.js

```diff
@@ -1 +1,14 @@
-module.exports = { extends: ['@commitlint/config-conventional'] };
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  plugins: ['commitlint-plugin-function-rules'],
+  rules: {
+    'scope-empty': [2, 'never'],
+    'function-rules/scope-enum': [
```

Comment:
How does this work? What the commits would need to look like?

Reply:
Wanna force the # as the organization descriptions says. <type>(<#scope>): <description>.